drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -166,7 +166,10 @@ def items_without_answers():
             items = copy.deepcopy(self.data.get('items', ''))
             for item in items:
                 del item['feedback']
-                del item['zone']
+                # Switching to `pop` from `del`, as we no longer have any guarantee of
+                # either `zone` or `zones` being present.
```

Comment:
Agreed; outside the context of "this is the change that we're making", the comment doesn't really make much sense.